README.md

```diff
@@ -1,5 +1,5 @@
-GitHub Action for Continuous Benchmarking
-=========================================
+# GitHub Action for Continuous Benchmarking
+
```

Comment:
I guess these are formatter changes. Would you revert the changes by your formatter and try to keep original format? If you feel it's hard (since some editor automatically formats text by default), please leave this document as-is. I can update this after merging this PR.